modules/process-testing/pages/process.adoc

```diff
@@ -0,0 +1,253 @@
+= Processes in integration tests
+:description: Manage process in integration tests using the Bonita test toolkit
+
+Learn to start and execute processes in your integration tests using the Bonita test toolkit.
+
+[NOTE]
+====
+Integrations tests are built upon the Bonita Test Toolkit, based on the open source https://github.com/bonitasoft/bonita-java-client[Bonita Java Client]. +
+The Bonita Test Toolkit is only available for Enterprise, Performance, Efficiency and Teamwork. editions. 
+====
+
+== Process definition
+
+A process definition represents an existing process deployed on the targeted runtime. It is used to get general information on the process, manage process instances and process parameters. 
+
+=== Retrieve a process definition
+
+A process definition can be retrieved using the Bonita test toolkit, the process name and optionally the process version. If the process version is not specified then the latest deployed version is retrieved (and not the latest version).
+
+[source, Java]
+----
+@BonitaTests(clearBDM = true, deleteProcessInstances = true)
+class ProcessIT {
+
+    private static final String PROCESS_NAME = "myProcess";
+    private static final String PROCESS_VERSION = "1.0.0";
+
+    @Test
+    void should_retrieve_process_definitions(BonitaTestToolkit toolkit) {
+        ProcessDefinition processDef1 = toolkit.getProcessDefinition(PROCESS_NAME); <1>
+        ProcessDefinition processDef2 = toolkit.getProcessDefinition(PROCESS_NAME,  PROCESS_VERSION); <2>
+        ...
+    }
+}
+----
+<1> The process is retrieved without specifying the version -> the latest deployed version is retrieved.
+<2> The process is retrieved is a given version.
+
+=== Retrieve process initiators
+
+The process initiators are the users that can start an instance of the process. They can be retrieved through the process definition.
+
+[source, Java]
+----
+@BonitaTests(clearBDM = true, deleteProcessInstances = true)
+class ProcessIT {
+
+    private static final String PROCESS_NAME = "myProcess";
+
+    @Test
+    void should_retrieve_process_initiators(BonitaTestToolkit toolkit) {
+        ProcessDefinition processDef = toolkit.getProcessDefinition(PROCESS_NAME); 
```

Comment:
```suggestion
        var myProcess= toolkit.getProcessDefinition( "myProcess"); 
```

Reply:
I prefer to not use var here, so users can know the type of the object returned by the method.